integration/access_log_test.go

```diff
@@ -458,54 +496,6 @@ func (s *AccessLogSuite) TestAccessLogFrontendWhitelist(c *check.C) {
 	checkNoOtherTraefikProblems(c)
 }
 
-func (s *AccessLogSuite) TestAccessLogAuthFrontendSuccess(c *check.C) {
```

Comment:
why is this test removed?

Reply:
Because of redundancy with the test L122, but i would agree to reintroduce it for clarity.

Reply:
if you are sure we are not losing anything in coverage, no need, thanks.